macOS/DuckDuckGo/VisualRefresh/VisualStyleConfigurable.swift

```diff
@@ -67,7 +149,7 @@ final class VisualStyleManager: VisualStyleConfigurable {
         overridesHandler.flagDidChangePublisher
             .filter { $0.0 == .visualRefresh }
             .sink { (_, enabled) in
-                /// Here I need to apply the visual changes. Should I restart the app?
+                /// Here I need to apply the visual changes. The easier way should be to restart the app.
```

Comment:
This (feature flag overriding) is an internal-only feature so it’s fine if it requires app restart. If you want, you can show an (unlocalized) alert from here stating that the user needs to restart the app for the changes to make effect. Perhaps add a “don’t show this again” checkbox. I suppose it may streamline the Ship Review, by reducing confusion :)

Reply:
Good idea. Let me create a follow-up task in my project to do this. I want to keep every PR slim for easier reviews